docs/CONTRIBUTING.adoc

```diff
@@ -133,6 +133,12 @@ If you are using Docker you can connect to the database using the following comm
 docker container exec -it plugin-health-scoring-db-1 psql -U <postgres-user-name>
 ----
 
+== Run the application on Gitpod
+
+This project includes a configuration in `.gitpod.yml` to be able to run on Gitpod. The required versions of Java and Maven are installed when starting the Gitpod workspace. Follow the steps to run the application using Docker compose.
```

Comment:
NOTE: Having one sentence per line simplify the correction / comments.

On the last line, could you elaborate a bit more (why would someone need to run the application? etc.)

```suggestion
This project includes a configuration in `.gitpod.yml` to be able to contribute to it on Gitpod.
The required versions of Java and Maven are installed when starting the Gitpod workspace.

Follow the steps to run the application using Docker compose.
```

Reply:
I have elaborated more in my latest commits and also tried to keep it concise